### PR TITLE
TFS - Converted to MacroAlmConnector

### PR DESCRIPTION
**Pull request by sibartlett**

### CODE REVIEW

**Comment by sibartlett** [comment]
@geoffwhittington any more feedback?

File: sdetools/modules/sync_tfs/tfs_plugin.py [hunk 1]
```diff
@@ -185,61 +200,31 @@ def _get_workitem_from_tfs(self, work_item_id, task_id):
...
-                'path': '/fields/System.Tags',
-                'value': tags_string
-            },
-        ]
+        create_args = {
+            'State': self.config['tfs_new_status'],
+            'Iteration Path': self.config['tfs_iterationpath'],
+            'Area Path': self.config['alm_project'],
+            'Work Item Type': self.config['tfs_issue_type'],
+            'Priority': self.translate_priority(task['priority']),
```
Comment (geoffwhittington) [inline sdetools/modules/sync_tfs/tfs_plugin.py]: this can be put into `TFS_DEFAULT_ISSUE_TEMPLATE `

**Comment by geoffwhittington** [comment]
I have 1 trivial amount of feedback, but I'm not attached to it. Merge away

**Comment by sibartlett** [comment]
Closes #383

**Pull request merged**